app/assets/images/payment_icons/mb.svg

```diff
@@ -1 +1 @@
-.893.894.895.895 0 01-.893-.893v-1.076a.199.199 0 11.397 0v1.076a.5.5 0 00.498.496.499.499 0 00.496-.497zM15.553 22.236a.2.2 0 010 .399h-.554a.748.748 0 01-.747-.747v-1.224a.199.199 0 01.397 0v1.222a.35.35 0 00.351.351h.553zM19.759 13.344a.746.746 0 11-1.482.157l-.613-5.618-2.159 5.494v.067a.733.733 0 01-.187.213l-.011.009a.722.722 0 01-.129.092l-.015.007-.012.007h-.074a.707.707 0 01-.218.041h-.054a.7.7 0 01-.244-.051H14.5l-.013-.007-.012-.007h-.007a.662.662 0 01-.117-.078l-.025-.022-.019-.018-.044-.012-.022-.024a.715.715 0 01-.079-.118v-.007l-.007-.011-.007-.013v-.055l-2.199-5.507-.611 5.618a.744.744 0 11-1.481-.157l.72-6.623a1.236 1.236 0 011.065-1.089h.04a1.49 1.49 0 01.189 0 1.23 1.23 0 011.091.779l1.846 4.665 1.842-4.68a1.226 1.226 0 011.092-.779c.063-.005.126-.005.189 0h.04a1.236 1.236 0 011.065 1.093zM12.202 22.413a.199.199 0 11-.395.047l-.17-1.434-.608 1.407a.2.2 0 01-.262.105.196.196 0 01-.103-.105l-.609-1.407-.171 1.434a.199.199 0 11-.395-.047l.2-1.665a.323.323 0 01.214-.265h.015a.339.339 0 01.4.158l.012.023.516 1.196.516-1.206a.33.33 0 01.293-.197h.061c.103.012.194.07.248.159a.212.212 0 01.027.057.174.174 0 01.011.039v.026zM16.421 22.436a.2.2 0 01-.399 0v-1.573h-.567a.2.2 0 010-.399h1.532a.2.2 0 010 .399h-.566zM17.664 22.436a.2.2 0 01-.399 0v-1.772a.2.2 0 11.399 0zM21.746 22.413a.2.2 0 01-.396.048l-.07-.563h-.685a.199.199 0 110-.397h.641l-.019-.147v-.014a.748.748 0 00-.064-.206.453.453 0 00-.403-.269.399.399 0 00-.19.046.559.559 0 00-.277.443l-.124 1.107a.198.198 0 01-.314.138.197.197 0 01-.08-.181l.124-1.107a.953.953 0 01.492-.755.845.845 0 01.176-.065.84.84 0 01.951.467c.027.054.05.111.068.169a1.274 1.274 0 01.034.18zM22.354 22.436a.2.2 0 11-.397 0v-1.642a.245.245 0 01.015-.099l.016-.035a.29.29 0 01.156-.13l.027-.007H22.245a.28.28 0 01.21.108l1.056 1.352v-1.319a.2.2 0 11.398 0v1.641a.277.277 0 01-.17.257l-.016.007a.283.283 0 01-.214-.011l-.025-.013a.319.319 0 01-.065-.057l-.007-.008-1.058-1.357zM25.464 22.236a.2.2 0 110 .399h-.586a.785.785 0 01-.785-.785v-.599a.787.787 0 01.785-.784h.586a.2.2 0 110 .398h-.586a.395.395 0 00-.275.118.384.384 0 00-.117.273v.599a.39.39 0 00.388.388h.587z" fill="#1d1d1b"/><path d="M26.617 20.862a.542.542 0 00-.377.15.487.487 0 00-.153.352v.371c0 .134.056.261.152.352a.542.542 0 00.377.15.538.538 0 00.376-.15.488.488 0 00.154-.352v-.371a.49.49 0 00-.153-.352.546.546 0 00-.377-.15m0-.398a.938.938 0 01.652.261.88.88 0 01.275.638v.371a.88.88 0 01-.275.638.941.941 0 01-1.304 0 .885.885 0 01-.275-.638v-.371a.883.883 0 01.275-.638.937.937 0 01.652-.261zM18.228 20.863v1.373h.769a.25.25 0 00.25-.251.253.253 0 00-.066-.169l-.008-.007a.248.248 0 00-.176-.075h-.265a.2.2 0 01-.173-.299.201.201 0 01.173-.099h.09a.242.242 0 00.15-.07.235.235 0 00-.164-.402h-.58zm-.399.674v-.865c0-.027.005-.054.016-.079a.245.245 0 01.046-.069.202.202 0 01.131-.056h.78c.351 0 .635.284.635.634a.64.64 0 01-.098.338.64.64 0 01.118.089l.012.014c.114.12.178.279.178.445a.65.65 0 01-.648.647h-.971a.199.199 0 01-.199-.199z" fill="#1d1d1b"/></svg>
```

Comment:
Can she shrink the icon a bit so there's spacing around it on all sides? It's quite close to the top/bottom edges.

Reply:
Sure. I have commited new file.